docs/model_control_mode.md

```diff
@@ -0,0 +1,68 @@
+# Model Control Mode
+
+TorchServe now supports model control mode with two settings "none"(default) and "enabled"
```

Comment:
Please add a short description here of what the two modes do and which APIs they disable/enable.